lighthouse-core/scripts/compare-runs.js

```diff
@@ -117,25 +117,32 @@ function round(value) {
  * @return {string}
  */
 function getProgressBar(i, total = argv.n * argv.urls.length) {
-  return new Array(Math.round(i * 40 / total)).fill('▄').join('').padEnd(40);
+  const bars = new Array(Math.round(i * 40 / total)).fill('▄').join('').padEnd(40);
+  return `${i} / ${total} [${bars}]`;
```

Comment:
the math here is off.  i is unique per URL and you want i+1 

here's a good repro: 

```sh
node lighthouse-core/scripts/compare-runs.js --name my-collectiondeleteme --gather -n 2 --urls https://www.example.com http://example.com
```

Reply:
I forgot to push this exact fix :(